app/presenters/flat_reasons_for_rejection_presenter.rb

```diff
@@ -54,6 +54,8 @@ def self.build_top_level_reasons(structured_rejection_reasons)
   end
 
   def self.y_n_to_boolean(string)
+    return nil unless %w[Yes No].include?(string)
```

Comment:
This better represents the value, rather than false.

This presenter seems to only be used in support, is there a reason I shouldn't move it into the support interface?

Reply:
Agree with the above, I think it makes sense to move it into support